### PR TITLE
Feature/question card

### PR DESCRIPTION
**Pull request by MonaHer**
Added question card including css (positioning)

### CODE REVIEW

**Comment by ArnelaKapllani** [comment]
Good Job!

File: styles.css [hunk 1]
```diff
@@ -15,5 +20,63 @@ border-color: hotpink;
```
Comment (AliaksandraNovikava) [inline styles.css:20]: How about creating a fixed header at the top of your page?
You could use e.g. `position: sticky;` and `top: 0;`.

File: styles.css [hunk 2]
```diff
@@ -15,5 +20,63 @@ border-color: hotpink;
...
+
+.answer-button {
+    padding: 5px;
+    margin: 40px auto 0 auto;
+    background-color: peachpuff;
+    border-color: hotpink;
+    color: hotpink;
+    font-size: 20px;
+}
+
```
Comment (AliaksandraNovikava) [inline styles.css:52]: How about changing the color of your answer-button on hover?
Smth like:
`.answer-button:hover { color: your-color; }`

**Review by AliaksandraNovikava — APPROVED** [review]
LGTM 👍 
Left you two small comments.

File: styles.css (hunk 1 above)
Comment (MonaHer) [reply styles.css:20]: Oh thank you so much, Alex! Great ideas, I will do that ! :)

**Pull request merged**